client/src/components/SearchResult.js

```diff
@@ -65,16 +75,15 @@ export const SearchResult = ({ resource, children, hideDefaults = false }) => {
           )}
           {resource.imported && (
             <>
-              <Link href={resource.url} as={`/resources/${resource.id}`}>
-                <Button
-                  label={`View on ${
-                    getReadable(resource.additional_metadata.import_source) ||
-                    'Source Site'
-                  }`}
-                  margin={{ bottom: 'small' }}
-                  primary
-                />
-              </Link>
+              <Button
+                as="a"
+                href={resource.url}
+                label={`View on ${
+                  getReadable(resource.import_source) || 'Source Site'
+                }`}
+                margin={{ bottom: 'small' }}
+                primary
+              />
```

Comment:
Why the change from Link to Button?

Reply:
We just didn't need the wrapper the `as="a"` does the same in terms of rendering but will also supply the button styles as per the designs. Also since this links out the an external url we don't need `next/router` to manage that